app/models/manageiq/providers/redhat/infra_manager/disk_attachment_builder.rb

```diff
@@ -5,7 +5,7 @@ def initialize(options = {})
     @name = options[:name]
     @thin_provisioned = BooleanParameter.new(options[:thin_provisioned])
     @bootable = BooleanParameter.new(options[:bootable])
-    @interface = options[:interface]
+    @interface = options[:interface] || ::Settings.ems.ems_ovirt.disk_attachment&.interface
```

Comment:
Not sure about a setting for this, if `"virtio-scsi"` is the default we can just do `|| "virtio-scsi"` and allow users to change it in the reconfigure UI.

Reply:
I think that currently the default is "VIRTIO" and set in the disk_attachment method 
https://github.com/ManageIQ/manageiq-providers-ovirt/blob/ed34bfcf8113703a09f32fc53d43c9606d4b4d85/app/models/manageiq/providers/redhat/infra_manager/disk_attachment_builder.rb#L11-L15

Maybe the default value can be changed there or remove from there a set in the constructor, as in this PR.
I agree with @agrare avoiding the setting if this value will be the default.

Reply:
So as per @gekorob comment, it seems that either we remove line 15 (https://github.com/ManageIQ/manageiq-providers-ovirt/pull/445/files/64363e16bf1a1207ebc86f73a84f1b2fd717559e#L15)
or we do not touch the constructor, I am ok either way.
I agree with @agrare that we do not need a setting for this (considering we all ready do it as a string in line 15 anyway).
@dsatsura thank you very much for your contribution! - can you please describe what scenario caused an issue where the interface was not set? I think this might shed some light on the ui question.

Reply:
Hi,
it's ok for me to set default via '|| "virtio_scsi"'.
I will update this PR and fix specs.

@borod108 disk interface always is set to **"virtio"**, no matter which **:interface** value you use in VmReconfigure request.